custom_components/button_plus/__init__.py

```diff
@@ -27,16 +28,17 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     hub = ButtonPlusHub(hass, device_configuration, entry)
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = hub
 
-    buttonplus_coordinator = ButtonPlusCoordinator(hass, hub)
+    # buttonplus_coordinator = ButtonPlusCoordinator(hass, hub)
 
-    await buttonplus_coordinator.async_config_entry_first_refresh()
+    # await buttonplus_coordinator.async_config_entry_first_refresh()
 
```

Comment:
If we are not going to use the Coordinator here then lets just remove it instead of commenting it out.

Reply:
Changed this back